go/consensus/tendermint/apps/scheduler/scheduler.go

```diff
@@ -487,10 +483,11 @@ func (app *schedulerApplication) electCommittee(
 	}
 
 	err = schedulerState.NewMutableState(ctx.State()).PutCommittee(ctx, &scheduler.Committee{
-		Kind:      kind,
-		RuntimeID: rt.ID,
-		Members:   members,
-		ValidFor:  epoch,
+		Kind:           kind,
+		RuntimeID:      rt.ID,
+		Members:        members,
+		ValidFor:       epoch,
+		LeaderInterval: leaderInterval,
```

Comment:
Yeah this can probably be removed as it just duplicates the configuration value from the runtime descriptor. Unless we would want the scheduler to be able to adjust this interval, but I don't see why we would need that.

Reply:
yeah will remove, there's a todo somewhere :)